src/runtime/offers-flow.js

```diff
@@ -116,11 +111,10 @@ export class OffersFlow {
       // so we need to check for oldSku to decide if it needs to be sent.
       // Otherwise we might accidentally block a regular subscription request.
       if (oldSku) {
-        new PayStartFlow(this.deps_, {
-          skuId: sku,
-          oldSku,
-          replaceSkuProrationMode: this.prorationMode,
-        }).start();
+        const skuSelectedResponse = new SkuSelectedResponse();
+        skuSelectedResponse.setSku(sku);
+        skuSelectedResponse.setOldSku(oldSku);
+        this.startPayFlow_(skuSelectedResponse);
```

Comment:
Doesn't this change mean that even if a publisher passes in a proration mode to the flow (which I realize currently isn't supported), offers-flow now has no way of passing it along?

Reply:
Yes, I removed it because this.prorationMode was always undefined. I can put it back with a TODO.

Reply:
That makes sense. So if we were to add that functionality in the future, we'd want to update skuSelectedResponse to have a proration mode field, right?

Reply:
Yeah, that sounds right